src/styles/_variables.scss

```diff
@@ -0,0 +1,23 @@
+$cell-size: 100px;
```

Comment:
move this file to utils folder.

More about file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)